pynitrokey/nk3/bootloader.py

```diff
@@ -61,7 +62,11 @@ def close(self) -> None:
 
     def reboot(self) -> None:
         if not self.device.reset(reopen=False):
-            raise Exception("Failed to reboot Nitrokey 3 bootloader")
+            # On Windows, this function returns false even if the reset was successful
+            if platform.system() == "Windows":
+                logger.warn("Failed to reboot Nitrokey 3 bootloader")
```

Comment:
nit: `logger.warn` is deprecated. Use `.warning` instead.

Reply:
thanks, fixed